packages/ecommerce-app-base/src/Editor/SortableComponent.tsx

```diff
@@ -57,28 +57,31 @@ export const SortableComponent: FC<Props> = ({
   const [productPreviews, setProductPreviews] = useState<Product[]>([]);
   const previousSkus = usePreviousSkus(skus);
 
-  const getProductPreviews = useCallback(
```

Comment:
removed `shouldRefetch` arg from this callback and reversed the dependency, so that consumers of this callback are responsible for determining `shouldRefetch`.  See [line 82](https://github.com/contentful/apps/pull/7944/files#diff-83200e90393b598c4e2f6fe5c314900c465f4d735d01a22a63fefa4f0c5b64edR82)